### PR TITLE
Polls 05 23

### PR DESCRIPTION
**Pull request by Hipogeo**
@TheExistence

 Outcomes for :  Investigate Deploying the MakerDAO Balance Sheet in ETH - May 23, 2022 still to be done.

### CODE REVIEW

File: governance/polls/Ratification Poll - Declaration of Intent - Invest in Short-Term Bonds (MIP13c3-SP12) - March 14, 2022.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ discussion_link: https://forum.makerdao.com/t/mip13c3-sp12-declaration-of-intent
 vote_type: Plurality Voting
 categories:
    - MIPs
+   - Impact-Medium
```
Comment (patrick-j-govalpha) [inline governance/polls/Ratification Poll - Declaration of Intent - Invest in Short-Term Bonds (MIP13c3-SP12) - March 14, 2022.md:8]: Don't change the old polls, it breaks the hash. Can you take this out?

File: governance/polls/Investigate Deploying the MakerDAO Balance Sheet in ETH - May 23, 2022.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
+title: Investigate Deploying the MakerDAO Balance Sheet in ETH - May 23, 2022
+summary: Signal your support or opposition to deploying the MakerDAO Balance Sheet in ETH.
+discussion_link: https://forum.makerdao.com/t/signal-request-deploy-the-balance-sheet-in-eth/14834
+vote_type: Plurality Voting
+categories:
+   - Collateral
+   - Impact-Medium
```
Comment (patrick-j-govalpha) [inline governance/polls/Investigate Deploying the MakerDAO Balance Sheet in ETH - May 23, 2022.md:8]: "Medium Impact" not "Impact-Medium"

File: governance/polls/Inclusion Poll - Implementation of a Institutional Vault from Antalpha - May 23, 2022.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+---
+title: Investigate Implementation of an Antalpha Institutional Vault - May 23, 2022
+summary: Signal your support or opposition to prioritizing the onboarding of an Antalpha Institutional Vault
+discussion_link: https://forum.makerdao.com/t/signal-request-antalpha-institutional-vault/15030
+vote_type: Plurality Voting
+categories:
+   - Collateral
```
Comment (patrick-j-govalpha) [inline governance/polls/Inclusion Poll - Implementation of a Institutional Vault from Antalpha - May 23, 2022.md:7]: add "Medium Impact"

**Review by patrick-j-govalpha — CHANGES_REQUESTED** [review]

File: governance/polls/Investigate Deploying the MakerDAO Balance Sheet in ETH - May 23, 2022.md [hunk 4]
```diff
@@ -25,11 +26,16 @@ The Governance Facilitators have placed a Governance Poll into the voting system
...
-Please review the discussion [thread](https://forum.makerdao.com/t/signal-request-deploy-the-balance-sheet-in-eth/14834) to help inform your position before voting.
+
+Please review the discussion [thread](https://forum.makerdao.com/t/signal-request-deploy-the-balance-sheet-in-eth/14834), including the [Towards taking asset risk](https://forum.makerdao.com/t/towards-taking-asset-risks-a-primer/14265) and [A balance sheet approach to an Aggressive Growth Strategy](https://forum.makerdao.com/t/a-balance-sheet-approach-to-executing-an-aggressive-growth-strategy-the-road-to-300m-revenues/14818) posts, to help inform your position before voting.
 
 ## Outcomes
 
- $TBD
+**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following action will be taken:**
+* The Mandated Actors will investigate a plan to offboard the above vault types from the Maker Protocol. 
+* A future on-chain poll will ask the community to implement a specific proposal from the Mandated Actors.
```
Comment (patrick-j-govalpha) [inline governance/polls/Investigate Deploying the MakerDAO Balance Sheet in ETH - May 23, 2022.md:36]: I think you misunderstood what I said on discord. Can you change this to:

**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following action will be taken:**
* The relevant Core Units will investigate options for deploying a portion of the MakerDAO balance sheet in ETH.
* Future on-chain polls will ask the community to support specific proposals.

**Review by patrick-j-govalpha — CHANGES_REQUESTED** [review]

File: governance/polls/Ratification Poll - Declaration of Intent - Invest in Short-Term Bonds (MIP13c3-SP12) - March 14, 2022.md (hunk 1 above)
Comment (Hipogeo) [reply governance/polls/Ratification Poll - Declaration of Intent - Invest in Short-Term Bonds (MIP13c3-SP12) - March 14, 2022.md]: Sorry. My bad. Done!

File: governance/polls/Investigate Deploying the MakerDAO Balance Sheet in ETH - May 23, 2022.md (hunk 2 above)
Comment (Hipogeo) [reply governance/polls/Investigate Deploying the MakerDAO Balance Sheet in ETH - May 23, 2022.md]: Done!

File: governance/polls/Inclusion Poll - Implementation of a Institutional Vault from Antalpha - May 23, 2022.md (hunk 3 above)
Comment (Hipogeo) [reply governance/polls/Inclusion Poll - Implementation of a Institutional Vault from Antalpha - May 23, 2022.md]: Done!

File: governance/polls/Investigate Deploying the MakerDAO Balance Sheet in ETH - May 23, 2022.md (hunk 4 above)
Comment (Hipogeo) [reply governance/polls/Investigate Deploying the MakerDAO Balance Sheet in ETH - May 23, 2022.md]: Fixed!

File: governance/polls/Inclusion Poll - Implementation of an Antalpha Institutional Vault- May 23, 2022.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+## Review
+
+This poll asks whether an Antalpha Institutional Vault should be prioritized for inclusion in the Maker Protocol by the [Core Units](https://mips.makerdao.com/mips/details/MIP38#mip38c2-core-unit-state) and [Domain Teams](https://mips.makerdao.com/mips/details/MIP7#mip7c2-the-current-domain-roles-list). 
+
+Please review the discussion [thread](https://forum.makerdao.com/t/signal-request-antalpha-institutional-vault/15030) to help inform your position before voting.
+
+## Outcomes
+
+**Regardless of the winning option the following actions will be taken at the conclusion of this poll:**
+* An Antalpha Institutional Vault vault-type will be added to the [Collateral Prioritisation Sheet](https://docs.google.com/spreadsheets/d/1IX9e2fyfz7djtDMKn5gMyGsyFxHoY75GncMbAjnSXrM/edit#gid=0) by the Collateral Engineering Services Core Unit.
```
[the remark this answers is not in the document]
Comment (Hipogeo) [reply governance/polls/Inclusion Poll - Implementation of an Antalpha Institutional Vault- May 23, 2022.md:34]: Done!

File: governance/polls/Inclusion Poll - Implementation of an Antalpha Institutional Vault- May 23, 2022.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+This poll asks whether an Antalpha Institutional Vault should be prioritized for inclusion in the Maker Protocol by the [Core Units](https://mips.makerdao.com/mips/details/MIP38#mip38c2-core-unit-state) and [Domain Teams](https://mips.makerdao.com/mips/details/MIP7#mip7c2-the-current-domain-roles-list). 
+
+Please review the discussion [thread](https://forum.makerdao.com/t/signal-request-antalpha-institutional-vault/15030) to help inform your position before voting.
+
+## Outcomes
+
+**Regardless of the winning option the following actions will be taken at the conclusion of this poll:**
+* An Antalpha Institutional Vault vault-type will be added to the [Collateral Prioritisation Sheet](https://docs.google.com/spreadsheets/d/1IX9e2fyfz7djtDMKn5gMyGsyFxHoY75GncMbAjnSXrM/edit#gid=0) by the Collateral Engineering Services Core Unit. 
+* This Antalpha Institutional Vault will be added with a 'greenlight score' calculated using the outcome of this poll using the formula `Score = 'Yes' Votes - 'No' Votes`
+* The Core Units and Domain Teams will use the Collateral Prioritisation Sheet to direct their work on onboarding new vault-types.
```
[the remark this answers is not in the document]
Comment (Hipogeo) [reply governance/polls/Inclusion Poll - Implementation of an Antalpha Institutional Vault- May 23, 2022.md:36]: Done!

File: governance/polls/Inclusion Poll - Implementation of an Antalpha Institutional Vault- May 23, 2022.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
+---
+title: Inclusion Poll - Implementation of an Antalpha Institutional Vault - May 23, 2022
```
Comment (patrick-j-govalpha) [inline governance/polls/Inclusion Poll - Implementation of an Antalpha Institutional Vault- May 23, 2022.md:2]:
```suggestion
title: Investigate Implementation of an Antalpha Institutional Vault - May 23, 2022
```

Comment (patrick-j-govalpha) [reply governance/polls/Inclusion Poll - Implementation of an Antalpha Institutional Vault- May 23, 2022.md:2]: Probably wouldn't bother with "Inclusion Poll" 

Add "Investigate" to the poll file name too @Hipogeo 

Also need to add a space " " after the word "Vault" in the file name

File: governance/polls/Inclusion Poll - Implementation of an Antalpha Institutional Vault- May 23, 2022.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+   - Collateral
+   - Medium Impact
+options:
+   0: Abstain
+   1: Yes (Greenlight)
+   2: No (Defer)
+start_date: 2022-05-23T16:00:00
+end_date: 2022-05-26T16:00:00
+---
+# Poll: Inclusion Poll - Implementation of an Antalpha Institutional Vault - May 23, 2022
```
Comment (patrick-j-govalpha) [inline governance/polls/Inclusion Poll - Implementation of an Antalpha Institutional Vault- May 23, 2022.md:16]:
```suggestion
# Poll: Investigate Implementation of an Antalpha Institutional Vault - May 23, 2022
```

File: governance/polls/Inclusion Poll - Implementation of an Antalpha Institutional Vault- May 23, 2022.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (patrick-j-govalpha) [reply governance/polls/Inclusion Poll - Implementation of an Antalpha Institutional Vault- May 23, 2022.md:34]: This isn't showing as done for me

File: governance/polls/Inclusion Poll - Implementation of an Antalpha Institutional Vault- May 23, 2022.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (patrick-j-govalpha) [reply governance/polls/Inclusion Poll - Implementation of an Antalpha Institutional Vault- May 23, 2022.md:36]: This isn't showing as done for me

File: governance/polls/Investigate Deploying the MakerDAO Balance Sheet in ETH - May 23, 2022.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Maker Community. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning Monday, May 23 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+This poll asks whether deploying the MakerDAO Balance Sheet in ETH should be prioritized for inclusion in the Maker Protocol by the [Core Units](https://mips.makerdao.com/mips/details/MIP38#mip38c2-core-unit-state) and [Domain Teams](https://mips.makerdao.com/mips/details/MIP7#mip7c2-the-current-domain-roles-list).
```
Comment (patrick-j-govalpha) [inline governance/polls/Investigate Deploying the MakerDAO Balance Sheet in ETH - May 23, 2022.md:27]:
```suggestion
This poll asks whether deploying the MakerDAO Balance Sheet in ETH should be prioritized for inclusion in the Maker Protocol by the [Core Units](https://mips.makerdao.com/mips/details/MIP38#mip38c2-core-unit-state).
```

Comment (patrick-j-govalpha) [reply governance/polls/Investigate Deploying the MakerDAO Balance Sheet in ETH - May 23, 2022.md:27]: Domain teams aren't really relevant to this so suggest taking that out

File: governance/polls/Investigate Deploying the MakerDAO Balance Sheet in ETH - May 23, 2022.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+summary: Signal your support or opposition to deploying the MakerDAO Balance Sheet in ETH.
+discussion_link: https://forum.makerdao.com/t/signal-request-deploy-the-balance-sheet-in-eth/14834
+vote_type: Plurality Voting
+categories:
+   - Collateral
+   - Medium Impact
+options:
+   0: Abstain
+   1: Yes (Greenlight)
+   2: No (Defer)
```
Comment (patrick-j-govalpha) [inline governance/polls/Investigate Deploying the MakerDAO Balance Sheet in ETH - May 23, 2022.md:12]:
```suggestion
   1: Yes
   2: No
```

This isn't a greenlight poll

File: governance/polls/Investigate Deploying the MakerDAO Balance Sheet in ETH - May 23, 2022.md [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+This poll asks whether deploying the MakerDAO Balance Sheet in ETH should be prioritized for inclusion in the Maker Protocol by the [Core Units](https://mips.makerdao.com/mips/details/MIP38#mip38c2-core-unit-state) and [Domain Teams](https://mips.makerdao.com/mips/details/MIP7#mip7c2-the-current-domain-roles-list). 
+
+
+Please review the discussion [thread](https://forum.makerdao.com/t/signal-request-deploy-the-balance-sheet-in-eth/14834), including the [Towards taking asset risk](https://forum.makerdao.com/t/towards-taking-asset-risks-a-primer/14265) and [A balance sheet approach to an Aggressive Growth Strategy](https://forum.makerdao.com/t/a-balance-sheet-approach-to-executing-an-aggressive-growth-strategy-the-road-to-300m-revenues/14818) posts, to help inform your position before voting.
```
Comment (patrick-j-govalpha) [inline governance/polls/Investigate Deploying the MakerDAO Balance Sheet in ETH - May 23, 2022.md:30]: This is good, lots of info for voters :)

**Review by patrick-j-govalpha — CHANGES_REQUESTED** [review]

File: governance/polls/Investigate Deploying the MakerDAO Balance Sheet in ETH - May 23, 2022.md (hunk 10 above)
Comment (Hipogeo) [reply governance/polls/Investigate Deploying the MakerDAO Balance Sheet in ETH - May 23, 2022.md:12]: Done!

File: governance/polls/Investigate Implementation of an Antalpha Institutional Vault - May 23, 2022.md [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+This poll asks whether an Antalpha Institutional Vault should be prioritized for inclusion in the Maker Protocol by the [Core Units](https://mips.makerdao.com/mips/details/MIP38#mip38c2-core-unit-state) and [Domain Teams](https://mips.makerdao.com/mips/details/MIP7#mip7c2-the-current-domain-roles-list). 
+
+Please review the discussion [thread](https://forum.makerdao.com/t/signal-request-antalpha-institutional-vault/15030) to help inform your position before voting.
+
+## Outcomes
+
+**Regardless of the winning option the following actions will be taken at the conclusion of this poll:**
+* An Antalpha Institutional Vault will be added to the [Collateral Status Index](https://docs.google.com/spreadsheets/d/1PDf_CzhGa7mLuOUfX6Bz3WrnCjDRhIjmu-vDZMMw4Qc/edit#gid=1077340672) by the Collateral Engineering Services Core Unit. 
+* This Antalpha Institutional Vault will be added with a 'greenlight score' calculated using the outcome of this poll using the formula `Score = 'Yes' Votes - 'No' Votes`
+* The Core Units and Domain Teams will use the Collateral Prioritisation Sheet to direct their work on onboarding new vault-types.
```
Comment (patrick-j-govalpha) [inline governance/polls/Investigate Implementation of an Antalpha Institutional Vault - May 23, 2022.md:36]:
```suggestion
* The Core Units and Domain Teams will use the Collateral Status Index to direct their work on onboarding new vault-types.
```

**Review by patrick-j-govalpha — CHANGES_REQUESTED** [review]
1 thing to change then it's good to go

**Review by patrick-j-govalpha — APPROVED** [review]
LGTM @prose11. Thanks, @Hipogeo and @TheExistence

**Pull request merged**